shared/metrics/src/main/java/io/pravega/shared/metrics/OpStatsData.java

```diff
@@ -19,6 +19,8 @@
  */
 public class OpStatsData {
     static final EnumSet<Percentile> PERCENTILESET = EnumSet.allOf(Percentile.class);
+    // Added for Micrometer's implementation, PERCENTILEARRAY MUST matches PERCENTILESET!
+    static final double[] PERCENTILEARRAY = {0.1, 0.5, 0.9, 0.99, 0.999, 0.9999};
```

Comment:
Minor: `PERCENTILEARRAY` -> `PERCENTILE_ARRAY` (same for `PERCENTILE_SET`)

Reply:
fixed